source/lib/JMSLab/builders/build_latex.py

```diff
@@ -47,7 +50,83 @@ def add_call_args(self):
                                   os.path.normpath(self.log_file))
         self.call_args = args
         return None
-     
+    
+    
+    def check_handout(self, target, env):
```

Comment:
@ew487 on the below, I guess maintaining parallelism with LyX is ideal if we can do it. If you see a reason why this isn't practical for the TeX builder let me know and we can figure out how best to compromise.

Thanks!

> Do you want to use the same kind of input control as we did for the lyx handout? ![image](https://private-user-images.githubusercontent.com/108754119/290924352-e7be9268-c1c2-4e25-9fba-f48aff9365cf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI2NzIwMjMsIm5iZiI6MTcwMjY3MTcyMywicGF0aCI6Ii8xMDg3NTQxMTkvMjkwOTI0MzUyLWU3YmU5MjY4LWMxYzItNGUyNS05ZmJhLWY0OGFmZjkzNjVjZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxNVQyMDIyMDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZjU4YWY0MDQzOThiOGFjMmQxZTA1MjA4NzdjNGE4ZTdkNjU3MWE3MjU4OWRmMmY1ODg3MzM3OGQ2MWJlY2ZjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.BHRMpeHL5ShUwPH7OaQIiMW1QpYfAJL0nb58TT8001c) I think this is causing some of the tests to fail...

Reply:
@jmshapir I think it would work practically! But I just wanted to check because from looking at the unit testing it seems like we're altering the expected behavior of the builder, so if someone based their project off of this template in the past, it might not work anymore?

Reply:
@ew487 thanks! To clarify, do you mean that the change we're considering here to the LaTeX builder will not be backwards compatible, in the sense that if someone adopts the new builder in an existing repository, they may need to update the syntax they use to call the builder?

If so, that sounds very likely fine. To help me think it through, can you give me an example of syntax that would work with the current builder (i.e. the one in `main`) but not in the alternative we're considering (here in #89)?

Reply:
To follow up here, while I think it's likely fine to change the syntax of the LaTeX builder, we probably _don't_ want to deprecate any functionality (e.g. bibtex etc.) Anyway we can figure it out once I have a better sense of the issues here thanks.

Reply:
We have not deprecated any functionality and are just requiring that the user enter target files using a specific syntax. So whereas before they are able to write `target = ["Template_tex.pdf", "Template_tex.log"], now we are enforcing that the second target file (if it exists) must be the path to the handout.

Per meeting, @jmshapir is fine with this so I will resolve the thread.